### ISSUE TITLE
Reproducing Results of the paper

### ISSUE THREAD

**Issue by proxymallick** [roywang021/EOD#1]
Hi Roy,
     Hope you are doing well! 
 I am finding it really hard to reproduce the results as that of the paper using the current codes. 
Also currently the codes take enormous amount of time to run, even with 8 GPU environment. Are these the right codes, or did you for forget to push the latest code?

Thanks

**Comment by roywang021** [comment]
Training shouldn't be too time consuming. Try to use the model weights provided in (https://huggingface.co/natewang021/eod/tree/main) and fine-tune with MSE_br loss. This approach should ensure quick and stable convergence.

**Comment by proxymallick** [comment]
Hi Roy,  Thank you very much for the swift reply!
Do you have log file which you can post for me to have a look at them?

Thank you
Regards
Prakash

**Comment by proxymallick** [comment]
[log.txt](https://github.com/roywang021/EOD/files/15397583/log.txt)

For VOC-COCO-20 , the results are 

[04/22 23:14:25] d2.evaluation.testing INFO: copypaste: mAP=13.84
[04/22 23:14:25] d2.evaluation.testing INFO: copypaste: WI=14.63
[04/22 23:14:25] d2.evaluation.testing INFO: copypaste: AOSE=4819.0
[04/22 23:14:25] d2.evaluation.testing INFO: copypaste: AP@K=55.71
[04/22 23:14:25] d2.evaluation.testing INFO: copypaste: P@K=46.19
[04/22 23:14:25] d2.evaluation.testing INFO: copypaste: R@K=65.89
[04/22 23:14:25] d2.evaluation.testing INFO: copypaste: AP@U=7.31
[04/22 23:14:25] d2.evaluation.testing INFO: copypaste: P@U=41.7
[04/22 23:14:25] d2.evaluation.testing INFO: copypaste: R@U=12.85

For your information I have attached my full log file! 
Any help will be highly appreciated.  Thanks heaps!

**Comment by roywang021** [comment]
The loss_clip was an experimental attempt that is now deprecated and should not be used. During the process of inference, the setting of score_thresh and uncertainty_thr will have a significant impact on the result. I apologize for any confusion; this work was conducted in my previous laboratory, and due to changes in the lab, I only saved the code and model weights without saving the log file. Loading the provided model weights directly should lead to improved results.
